instance.tf

```diff
@@ -42,10 +44,20 @@ resource "aws_security_group" "allow_tls" {
 
   ingress {
     description = "TLS from VPC"
-    from_port   = 80
-    to_port     = 80
+    from_port   = 0
+    to_port     = 65535
     protocol    = "tcp"
-    cidr_blocks = [aws_vpc.main_vpc.cidr_block]
+    /*cidr_blocks = [aws_vpc.main_vpc.cidr_block]*/
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  ingress {
+    description = "TLS from VPC"
+    from_port   = 0
+    to_port     = 65535
+    protocol    = "tcp"
+    /*cidr_blocks = [aws_vpc.main_vpc.cidr_block]*/
+    cidr_blocks = ["0.0.0.0/0"]
```

Comment:
What is the reason you are changing the security group rule? I believe the previous configuration is fine and this change is not needed